docs/tutorials/native-vrf/commit-reveal-cadence.md

```diff
@@ -19,21 +19,19 @@ keywords:
 
 # Secure Randomness with Commit-Reveal in Cadence
 
-Randomness is a critical component in blockchain applications, enabling fair and unpredictable outcomes for use cases like gaming, lotteries, and cryptographic protocols. The most basic approach to generating a random number on EVM chains is to utilize block hashes, which combines the block hash with a user-provided seed and hashes them together. The resulting hash can be used as a pseudo-random number. However, this approach has limitations:
-
-1. Predictability: Miners can potentially manipulate the block hash to influence the generated random number.
-2. Replay attacks: In case of block reorganizations, the revealed answers will not be re-used again.
```

Comment:
points 1. and 2. are actually the same (miners reordering transactions within a block, choosing what transaction they add/remove, even adding their own transactions in the block and varying the text inside the transaction to update the block hash - all this is what we refer to as source manipulation or biasing the source)

Reply:
Biasing is actually the problem rather then predictability so I removed the mention of predictability (these terms are more explained in depth in this [forum post](https://forum.flow.com/t/secure-random-number-generator-for-flow-s-smart-contracts/5110))